content/en/docs/languages/python/libraries.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Using instrumentation libraries
+linkTitle: Libraries
+weight: 40
+cSpell:ignore: HTTPX httpx instrumentor uninstrument
+---
+
+{{% docs/languages/libraries-intro "Python" %}}
+
+## Use instrumentation libraries
+
+If a library isn't included with OpenTelemetry, you can use
+[instrumentation libraries](/docs/specs/otel/glossary/#instrumentation-library)
+to generate telemetry data for a library or framework.
+
+For example,
+[the instrumentation library for HTTPX](<[https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNetCore](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation/opentelemetry-instrumentation-httpx)>)
+automatically creates [spans](/docs/concepts/signals/traces/#spans) based on
+HTTP requests.
+
+## Setup
+
+You can install each instrumentation library separately using pip. For example:
+
+```sh
+pip install opentelemetry-instrumentation-{integration}
+```
+
+In the previous example, `{integration}` is the name of the instrumentation. For
+a complete list of instrumentations and their names, browse the
+[instrumentation](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation)
+directory of the `opentelemetry-python-contrib` repository on GitHub.
```

Comment:
that seems redundant as we have a chapter below that is called "available instrumentation libraries"